freezeyt/freezer.py

```diff
@@ -59,11 +59,14 @@ async def freeze_async(app, config):
 }
 
 
-def check_mimetype(url_path, headers, default='application/octet-stream'):
+def check_mimetype(
+    url_path, headers,
+    frecognizer=guess_type, default='application/octet-stream'
+):
     if url_path.endswith('/'):
         # Directories get saved as index.html
         url_path = 'index.html'
-    file_type, file_encoding = guess_type(url_path)
+    file_type, file_encoding = frecognizer(url_path)
     if not file_type:
         file_type = default
```

Comment:
When `None` is returned for `file_type`, the default is used. This should be in the documentation and tests.

Reply:
- [ ] documentation
- [ ] tests